app/characterization_services/identify_file_characterization_service.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+# Class for ImageMagick identify based file characterization service
+class IdentifyFileCharacterizationService < TikaFileCharacterizationService
+  attr_reader :mime_types, :file_node, :persister
+
+  def initialize(file_node:, persister:)
+    @file_node = file_node
+    @persister = persister
+    @mime_types = {
+      "TIFF": "image/tiff"
```

Comment:
This isn't great, but that is the only mime type we really support right now.  We could add others as we add support for them?

Reply:
You can get a real mime type from the verbose output of identify. See #559.